addons/rating/static/src/messaging_menu_patch.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-inherit="mail.MessagingMenu.content" t-inherit-mode="extension">
-        <xpath expr="//NotificationItem[@body='preview.message?.inlineBody']" position="attributes">
-            <attribute name="rating">preview.message?.rating</attribute>
-            <attribute name="onClick">(isMarkAsRead) => preview.message?.rating and !isMarkAsRead ? this.openThread(preview.message) : this.onClickPreview(isMarkAsRead, preview)</attribute>
+        <xpath expr="//NotificationItem[@body='message?.inlineBody']" position="attributes">
+            <attribute name="rating">message?.rating</attribute>
+            <attribute name="onClick">(isMarkAsRead) => message?.rating and !isMarkAsRead ? this.openThread(thread) : this.onClickThread(isMarkAsRead, thread)</attribute>
```

Comment:
`openThread` now taking a thread rather than a message?

If that is indeed incorrect and runbot is green, it needs another test to cover it.

Reply:
This was a mistake, there was no test coverage